Maths/CheckPalindrome.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * This function checks if given number is palindromic
+ * e.g. 121
+ * @param integer $input
+ * @return boolean
+ */
+function isPalindrome(int $input): bool
```

Comment:
This function name will need to be changed to a different name.  The same function name is declared in `Strings/CheckPalindrome.php`, so there's a collision when running the tests.  Maybe you could name it to `isPalindromic` or `isNumericPalindrome`.